docs/ballerina-by-example/examples/grpc-bidirectional-streaming/client/chat.sample.client.bal

```diff
@@ -0,0 +1,56 @@
+// This is client implementation for bidirectional streaming scenario
+package client;
+import ballerina/net.grpc;
```

Comment:
It is better to separate the package declaration and import declarations with a newline.

Reply:
Fixed in f2b7cd26cb71721658fbc018e48d1aecaddaecd2